string file_path
string file_name
bool reset_counter
---
bool result
